package.json

```diff
@@ -57,6 +68,7 @@
     "@esri/hub-sites": "^12.3.2",
     "@esri/hub-teams": "^9.32.1",
     "@types/lodash": "^4.14.171",
+    "coverage-badges-cli": "^1.0.11",
```

Comment:
My fault again. I  installed `coverage-badges-cli` package as dependencies instead of dev dependencies. Could you please add this package as dev dependencies instead?